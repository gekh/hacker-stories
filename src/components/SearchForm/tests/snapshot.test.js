import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import { SearchForm } from '..';


describe('SearchForm', () => {
  const searchFormProps = {
    searchTerm: 'React',
    onSearchInput: jest.fn(),
    onSearchSubmit: jest.fn().mockImplementation(e => e.preventDefault()),
  };

  it('renders a snapshot', () => {
    const { container } = render(<SearchForm {...searchFormProps} />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
