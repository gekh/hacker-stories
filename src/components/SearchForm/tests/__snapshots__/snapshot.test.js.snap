// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchForm renders a snapshot 1`] = `
<form
  class="sc-jSUZER mefOQ"
>
  <label
    class="sc-hLBbgP gcKPTk"
    for="search"
  >
    <strong>
      Search:
    </strong>
  </label>
  Â 
  <input
    class="sc-eDvSVe genYmP"
    id="search"
    type="text"
    value="React"
  />
  <button
    class="sc-bcXHqe sc-dkrFOg cctTZy bGecVM"
    type="submit"
  >
    Submit
  </button>
</form>
`;
